pkg/virt-handler/vm.go

```diff
@@ -1576,6 +1580,17 @@ func sshRelatedCommandsSupported(commands []v1.GuestAgentCommandInfo) bool {
 
 func calculatePausedCondition(vmi *v1.VirtualMachineInstance, reason api.StateChangeReason) {
 	switch reason {
+	case api.ReasonPausedMigration:
+		log.Log.Object(vmi).V(3).Info("Adding paused condition")
+		now := metav1.NewTime(time.Now())
```

Comment:
supernit: we can initiate this variable outside of the switch clause and just use it for every case